pkg/action/validate.go

```diff
@@ -42,7 +52,35 @@ func existingResourceConflict(resources kube.ResourceList) error {
 			return errors.Wrap(err, "could not get information about the resource")
 		}
 
-		return fmt.Errorf("existing resource conflict: namespace: %s, name: %s, existing_kind: %s, new_kind: %s", info.Namespace, info.Name, existing.GetObjectKind().GroupVersionKind(), info.Mapping.GroupVersionKind)
+		// Allow adoption of the resource if it already has app.kubernetes.io/instance and app.kubernetes.io/managed-by labels.
```

Comment:
the annotations listed above don't match up to the comment provided here. Does the comment here need to be updated?

Reply:
ah, yep that's a holdover from the initial implementation